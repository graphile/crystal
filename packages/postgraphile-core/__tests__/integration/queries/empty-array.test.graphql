query {
  allPeople(first: 1) {
    nodes {
      id
      name
      aliases
    }
  }
}
