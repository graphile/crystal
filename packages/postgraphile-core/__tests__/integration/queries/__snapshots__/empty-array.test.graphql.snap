// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "allPeople": Object {
      "nodes": Array [
        Object {
          "aliases": Array [],
          "id": 1,
          "name": "John Smith",
        },
      ],
    },
  },
}
`;
